app/move/controllers/create/base.js

```diff
@@ -24,6 +27,48 @@ class CreateBaseController extends FormWizardController {
 
     next()
   }
+
+  async locals(req, res, callback) {
+    try {
+      const currentLocation = req.session.currentLocation
+      const {
+        date,
+        first_names: firstNames,
+        last_name: lastName,
+        time_due: timeDue,
+        move_type: moveType,
+        to_location: toLocation,
+        additional_information: additionalInfo,
+      } = req.sessionModel.toJSON()
+      let toLocationDetail
+
+      if (toLocation) {
+        toLocationDetail = await referenceDataService.getLocationById(
+          toLocation
+        )
+      }
+
+      const moveSummary = presenters.moveToMetaListComponent({
```

Comment:
I'd be tempted to pass the session model in here as the field names should match the expected values of the presenter?

Saves re-writing it twice.

You will need to override the `to_location` property though to be what the reference data service returns. But that could be done using spread.